packages/reputation-miner/ReputationMinerClient.js

```diff
@@ -95,8 +95,52 @@ class ReputationMinerClient {
         return res.status(200).send({ active: activeAddr, inactive: inactiveAddr });
       });
 
+      // Query users who have given reputation in colony
+      this._app.get("/:rootHash/:colonyAddress/:skillId/", async (req, res) => {
+        if (
+          !ethers.utils.isHexString(req.params.rootHash) ||
+          !ethers.utils.isHexString(req.params.colonyAddress) ||
+          !ethers.utils.bigNumberify(req.params.skillId)
+        ) {
+          return res.status(400).send({ message: "Requested reputation does not exist or invalid request" });
```

Comment:
Seems as though you could be more specific with the error messages here. In this case, the request is invalid, while in the next case, the reputation does not exist. Using the same message for both just withholds information from the user.

Reply:
I agree this one could be better. I don't believe the reputation not existing would trigger the next one, though - should just return an empty list. I guess a failed `try/catch` should return a 500 error code though. Have tried to improve the error messages.